readme.md

```diff
@@ -28,3 +28,40 @@ This installer contains the base docker images needed to perform a full installa
     ```
 
 Then visit [your local install](https://localhost:8484/) in a webrowser.
+
+## Full Text Search
+
+### Create _tsearch_ table
+
+Below is an example of creating _tsearch_ table for the full text search.
+
+```bash
+docker exec package_gmf_db psql -c "
+    INSERT INTO main.tsearch (the_geom, layer_name, label, public, role_id, lang, ts)
+    SELECT
+        ST_TRANSFORM(wkb_geometry, 2056),
```

Comment:
In this project we use the projection webmecator : `3857` instead of the swiss `2056`